test/data/nightfall_test_config.json

```diff
@@ -2,5 +2,6 @@
     "detectors": {
         "CREDIT_CARD_NUMBER": "POSSIBLE",
         "PHONE_NUMBER": "LIKELY"
-    }
+    },
+    "tokenExclusionList": ["4242-4242-4242-4242", "xG0Ct4Wsu3OTcJnE1dFLAQfRgL6b8tIv"]
```

Comment:
I guess the nightfalldlp annotation is showing up since we are using the action and not the code in this diff